atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/DbKvs.java

```diff
@@ -96,7 +98,11 @@ public static DbKvs create(DbKeyValueServiceConfig config) {
         ReentrantManagedConnectionSupplier connSupplier = new ReentrantManagedConnectionSupplier(connManager);
         SqlConnectionSupplier sqlConnSupplier = getSimpleTimedSqlConnectionSupplier(connSupplier);
 
-        return new DbKvs(config, config.tableFactorySupplier().get(), sqlConnSupplier);
+        DbKvs dbKvs = new DbKvs(config, config.tableFactorySupplier().get(), sqlConnSupplier);
+
+        dbKvs.init();
+
+        return dbKvs;
```

Comment:
Internal user of dbkvs may not call DbKvs.create() and thus miss initialization code.  This shouldn't be a problem for the current initialization code here (in fact it happens to be somewhat of a feature in this case), but we need to make sure to remember this in the future (so internal uses don't end up borked in certain code paths).